api/index.js

```diff
@@ -24,6 +25,7 @@ const apiRequest = (
     method,
     credentials: 'include',
     headers: requestHeaders,
+    body: data && JSON.stringify(data),
```

Comment:
If you add this feature, I recommend you to check the data type.

If `data` is an Object, we should stringify it to JSON, if text/number, only inline it. Also, based on data type we have to preset `Content-Type` header to either `application/json` or `text/plain` unless user defines it themselves.